sdk-utils/src/main/kotlin/com/rakuten/tech/mobile/sdkutils/eventlogger/Event.kt

```diff
@@ -19,32 +19,39 @@ internal data class Event(
     val rmcSdks: Map<String, String>? = null,
     val info: Map<String, String>? = null,
     var occurrenceCount: Int = 0,
-    @SerializedName("firstOccurrenceOn") var firstOccurrenceMillis: Long? = null
-) {
-    val eventVersion = "1.0"
-    val platform = Platform.ANDROID.displayName
-}
+    val eventVersion: String = "1.0",
+    val platform: String = "Android",
+    @SerializedName("firstOccurrenceOn") val createdOn: Long = System.currentTimeMillis()
+)
 
 // ---------------------------- Event enums ----------------------------
 internal enum class EventType(val displayName: String) {
     CRITICAL("0"),
     WARNING("1")
 }
 
-internal enum class Platform(val displayName: String) {
```

Comment:
reason behind this change => since for this platform will always be "Android" and no scope for any other value, or something else ?

Reply:
No specific reason, just removed unnecessary declarations

Reply:
Okay, It makes sense.